### PR TITLE
Format and Sort features in `Cargo.toml` files

### PR DESCRIPTION
**Pull request by ToufeeqP**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- [ ] Feature
- [ ] Bugfix
- [ ] Refactor
- [x] Format
- [ ] Documentation
- [ ] Testing
- [x] Other: CI update

## Checklist
- [x] I have performed a self-review of my own code.
- [x] The tests pass succesfully with `cargo test`.
- [x] The code was formatted with `cargo fmt`.
- [x] The code compiles with no new warnings with `cargo build --release` and `cargo build --release --features runtime-benchmarks`.
- [x] The code has no new warnings when using `cargo clippy`.
- [x] If this change affects documented features or needs new documentation, I have created a PR with a [documentation update](https://github.com/availproject/availproject.github.io).

### CODE REVIEW

**Review by fmiguelgarcia — COMMENTED** [review]
Could you pls add the format checker to the CI?
Are you using `dprint` for this?

**Comment by ToufeeqP** [comment]
> Could you pls add the format checker to the CI? Are you using `dprint` for this?

I am using Zepter, which parity also uses. Was thinking of adding it to the formatting but wasn't sure

**Review by markopoloparadox — APPROVED** [review]

**Comment by fmiguelgarcia** [comment]
> s thinking of adding it to the formatting bu

**Comment by fmiguelgarcia** [comment]
> > Could you pls add the format checker to the CI? Are you using `dprint` for this?
> 
> I am using [Zepter](https://lib.rs/crates/zepter), which parity also uses. Was thinking of adding it to the formatting but wasn't sure, I will add it now

In `avail-core` we are using `dprint` which is also integrated in its CI, and format also `JSON` & `MarkDown` files.
We should unify both repos.

**Comment by ToufeeqP** [comment]
> > > Could you pls add the format checker to the CI? Are you using `dprint` for this?
> > 
> > 
> > I am using [Zepter](https://lib.rs/crates/zepter), which parity also uses. Was thinking of adding it to the formatting but wasn't sure, I will add it now
> 
> In `avail-core` we are using `dprint` which is also integrated in its CI, and format also `JSON` & `MarkDown` files. We should unify both repos.

Ok, I didn't noticed that we already have a toml formatting tool integrated. And, I agree that we should use the single one in both repos. so shall we use `dprint` only then?

File: .github/workflows/default.yml [hunk 1]
```diff
@@ -45,6 +45,22 @@ jobs:
...
           args: --manifest-path avail-subxt/Cargo.toml --check
 
+      - name: Install zepter
+        continue-on-error: true
+        run: |
+          cargo install zepter --locked --version 0.11.0 -q -f --no-default-features && zepter --version
+
+      - name: Check Avail toml formatting
+        # Not enforcing the toml formatting for now
+        continue-on-error: true
```
Comment (fmiguelgarcia) [inline .github/workflows/default.yml:55]: Not enforcing?

Comment (ToufeeqP) [reply .github/workflows/default.yml:55]: Yeah, the check is just for awareness purposes now, later maybe we can enforce it, wdyt? just to not annoy the devs on toml formatting

Comment (fmiguelgarcia) [reply .github/workflows/default.yml]: IMHO we should enforce it. Otherwise we will need a periodically task to apply formatting.

**Review by fmiguelgarcia — APPROVED** [review]

**Comment by ToufeeqP** [comment]
> @ToufeeqP What do you think about moving lint related checks into a new workflow? Should make it easier to reason with what our workflows are actually doing.

It makes sense to me

**Pull request merged**